clients/node/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@ministryofjustice/hmpps-non-associations-api",
+  "version": "0.0.0",
+  "private": false,
+  "description": "HMPPS Non-associations API REST client",
+  "author": {
+    "name": "HMPPS Digital (MoJ Digital & Technology)",
+    "email": "info@digital.justice.gov.uk",
+    "url": "https://mojdigital.blog.gov.uk/"
+  },
+  "homepage": "https://github.com/ministryofjustice/hmpps-non-associations-api",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ministryofjustice/hmpps-non-associations-api.git"
+  },
+  "license": "MIT",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "/dist"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "clean": "rm -rf ./dist/",
+    "format": "prettier --write src test",
+    "lint": "eslint . --cache --max-warnings 0",
+    "test": "jest"
+  },
+  "engines": {
+    "node": "18 || 20"
```

Comment:
Could also be looser? I'd imagine we don't care as it's more of a client project choice, e.g. I'd get anything > 18 would do?

Unless we think newer version of Node may break this package somehow?

Reply:
i just happen to have added testing on v18 (what we all currently use, except dps) and v20 which is about to go LTS (and i figure we will all move to it shortly)